### PR TITLE
[Refine #76] Avoid storing test data in repo.

### PR DESCRIPTION
**Pull request by M157q**
ref #76

### CODE REVIEW

File: zdict/tests/dictionaries/test_spanish.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from zdict.dictionaries.spanish import SpanishDict
+from zdict.exceptions import NotFoundError
+
+
+class TestSpansishDict:
+    d = SpanishDict()
+    word = 'Spanish'
+    timeout = 5
+    raw_html = d._get_raw(word, timeout)
+    record = d.query(word, timeout)
```
Comment (iblis17) [inline zdict/tests/dictionaries/test_spanish.py]: It seems that there is no function requires it but `test_show`.
How about just setup it as local variable of `test_show`?

Comment (iblis17) [inline zdict/tests/dictionaries/test_spanish.py]: Ok, i found that `test_query_normal` require it.
Ignore previous comment.

Comment (iblis17) [inline zdict/tests/dictionaries/test_spanish.py]: I will suggest that pust line 9-13 in a setup/teardown method.
ref: https://pytest.org/latest/xunit_setup.html#method-and-function-level-setup-teardown
e.g.
```python
class TestSpansisDict:
  def setup_method(self, method):
     self.d = SpanihDict()
      ...
  def teardown_method(self, method):
     del self.d
     del ...
```
This snippet can ensure our each testing method not to be effect by other else testing method. (or say isolation our testing method).
I think those vars (`self.d`, `self.record` ... etc) can consider as kind of fixture, so prepare them in `setup` methd will be resonable.

**Comment by M157q** [comment]
`content` in `test_jisho.py` also needs this refinement.

**Comment by M157q** [comment]
https://pytest.org/latest/xunit_setup.html#class-level-setup-teardown

**Pull request merged**